http-gateway/swagger.yaml

```diff
@@ -261,6 +295,19 @@ components:
           authorizationUrl: https://auth.plgd.cloud/oauth/token
           scopes:
             openid: to authorize access to a user's details
+  requestBodies:
+    WSAccessToken:
```

Comment:
It's obvious that it's websocket body, as you are using it in websocket method. Don't use technology prefixes in model names. Just name it based on purpose it serves. e.g. `AuthorizationContext` ? Which can be at the end send at any point during opened connection.

Reply:
Fixed